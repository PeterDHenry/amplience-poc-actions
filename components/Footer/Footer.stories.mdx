import { Meta, Story, Canvas } from '@storybook/addon-docs/blocks'
import Footer from './Footer'

<Meta title="Layout/Checkout Footer" component={Footer} />

# Footer

The website Footer

<Canvas>
  <Story name="Default">
    <Footer />
  </Story>
</Canvas>
