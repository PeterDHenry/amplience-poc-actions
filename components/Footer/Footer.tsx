import React from 'react'
import styled from 'styled-components'

const Foot = styled.footer`
  background-color: #e7e7e7;
`

export default function Footer() {
  return (
    <Foot className="text-center p-5" data-page-area="Footer">
      <p className="text-secondary">
        <a
          data-ga-label="Terms &amp; conditions"
          href="/h/option/terms_conditions?isInLightBox=True&amp;isCheckout=True"
          className=""
        >
          Terms &amp; conditions
        </a>
      </p>
      <small className="text-center">
        Â© 2021 Wiggle Ltd. 1000 Lakeside, Suite 310, Third Floor N E Wing,
        Portsmouth, Hampshire, PO6 3EN, United Kingdom
        <br />
        Company No: 2667809 | VAT No: GB 884 2571 94
      </small>
    </Foot>
  )
}
