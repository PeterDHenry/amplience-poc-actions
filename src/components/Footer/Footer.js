import React from 'react';

const Footer = () => (
  <div className="bg-gray-800 p-5 row-auto text-white mt-20">
    Â© Wiggle Ltd. 2021.<br />
    Wiggle Ltd. 1000 Lakeside, Suite 310, Third Floor N E Wing, Portsmouth, Hampshire, PO6 3EN, United Kingdom<br />
    VAT No: GB 884 2571 94 | Company No: 02667809
  </div>
);
export default Footer;
