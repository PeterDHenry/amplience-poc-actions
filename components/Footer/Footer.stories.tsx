import { Meta, Story } from '@storybook/react'
import { Footer as Component } from './Footer'

const meta: Meta = {
  title: 'Layout/Checkout',
  component: Component,
  argTypes: {},
}

export default meta

export const Footer: Story = () => <Component />
