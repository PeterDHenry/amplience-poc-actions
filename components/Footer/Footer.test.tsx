import { render } from '@testing-library/react'
import Footer from './Footer'

describe('Footer component', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Footer />, {})
    expect(asFragment()).toMatchSnapshot()
  })

  it('should render footer element', () => {
    const { container } = render(<Footer />)
    expect(container.getElementsByTagName('footer')).toBeTruthy()
  })
})
