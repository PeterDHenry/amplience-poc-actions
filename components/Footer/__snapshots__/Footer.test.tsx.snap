// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component matches snapshot 1`] = `
<DocumentFragment>
  <footer
    class="Footer__Foot-sc-105w7nn-0 sQZAk text-center p-5"
    data-page-area="Footer"
  >
    <p
      class="text-secondary"
    >
      <a
        class=""
        data-ga-label="Terms & conditions"
        href="/h/option/terms_conditions?isInLightBox=True&isCheckout=True"
      >
        Terms & conditions
      </a>
    </p>
    <small
      class="text-center"
    >
      Â© 2021 Wiggle Ltd. 1000 Lakeside, Suite 310, Third Floor N E Wing, Portsmouth, Hampshire, PO6 3EN, United Kingdom
      <br />
      Company No: 2667809 | VAT No: GB 884 2571 94
    </small>
  </footer>
</DocumentFragment>
`;
